components/Person.js

```diff
@@ -15,111 +15,92 @@ const Container = styled.div`
   }
 `;
 
+const AvatarContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  justify-self: center;
+  align-self: center;
+`;
+
+const LogoNetwork = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+  align-self: center;
+  margin-left: 20px;
+  margin-right: 20px;
+
+  @media (min-width: 768px) {
+    flex-direction: row;
+  }
+`;
+
+const Description = styled.p`
+  font-family: "Montserrat";
+  font-size: 15px;
+  width: 80%;
+  color: #ffffff;
+  letter-spacing: 3px;
```

Comment:
It's quite discouraged to randomly change `letter-spacing` here and there, it makes the font looks inconsistent.